producer.go

```diff
@@ -71,6 +71,7 @@ type ProducerConfig struct {
 	Partitioner           PartitionerConstructor
 	KeyEncoder            Encoder
 	ValueEncoder          Encoder
+	AckSuccesses			bool
 
 	//Retries            int //TODO ??
```

Comment:
retries should go here yes after https://github.com/stealthly/siesta/issues/2 is done